spec/tic_tac_toe_spec.rb

```diff
@@ -1,14 +1,75 @@
 # frozen_string_literal: true
 
-# require 'bin/main'
-require './lib/player'
+load File.expand_path('../bin/main', __dir__)
+require File.expand_path('../lib/player', __dir__)
+require File.expand_path('../lib/board', __dir__)
```

Comment:
- [ ]  Using `require_relative` for files in your directory instead of `require` is a ruby convention.

"Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."